app/src/main/java/org/fossasia/openevent/general/search/SearchLocationFragment.kt

```diff
@@ -31,22 +31,26 @@ class SearchLocationFragment : Fragment() {
     private val geoLocationViewModel by viewModel<GeoLocationViewModel>()
     private val safeArgs: SearchLocationFragmentArgs by navArgs()
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    private val AUTOCOMPLETE_FRAG_TAG = "AutoComplete_Frag"
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
         rootView = inflater.inflate(R.layout.fragment_search_location, container, false)
 
         val thisActivity = activity
         if (thisActivity is AppCompatActivity) {
-            thisActivity.supportActionBar?.show()
-            thisActivity.supportActionBar?.title = ""
-            thisActivity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+            thisActivity.supportActionBar?.hide()
```

Comment:
Here you hide it in activity, not in the fragment, so you need to show it again in search fragment.

Reply:
Hmm , that for removing actionbar in SearchLocationFragment.kt. 
Will have to enable the actionbar in onResume() of SearchFragment.kt 
Thanks for pointing it out :) .will do it

Reply:
Not onReasume, show it in onDestroyView of search location fragment.

Reply:
Yeah sorry . I forgot these are fragments. Will have to call  this in `onDestroyView` .